File: src/main/java/tri/le/music/entity/base/HumanEntity.java
package tri.le.music.entity.base;

/**
 * Created by TriLe on 3/12/2016.
 */
public class HumanEntity extends PublicInfoEntity {

    private String name;

    private String description;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: src/main/java/tri/le/music/entity/HumanEntity.kt
package tri.le.music.entity

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

/**
 * Created by TriLe on 2016-04-21.
 */
@Entity
class HumanEntity(
        var name: String = "",

        var description: String = "",

        var url: String = "",

        var imageUrl: String = "",

        var type: HumanType = HumanType.Singer
) {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    var id: Long = 0

    enum class HumanType {
        Composer, Singer
    }
}